stable/airflow/templates/deployments-scheduler.yaml

```diff
@@ -158,6 +158,8 @@ spec:
               sleep 10 &&
               mkdir -p /usr/local/airflow/.local/bin &&
               export PATH=/usr/local/airflow/.local/bin:$PATH &&
+              echo 'installing requirements...' &&
```

Comment:
I don't understand this part is already in the source tree of 3.0.2

https://github.com/helm/charts/blob/master/stable/airflow/templates/deployments-scheduler.yaml#L136

Reply:
Seems I created pull request earlier and resolved conflicts since that moment only today. I completely stuck with resolving conflicts, could you please help me?

Reply:
It is simpler to just restart over.. It is just a few lines :)

Reply:
The problem, that I have 3 pending contributions into stable/airflow, every contrib requires a new version of the Chart. They are not approved instantly and I don't know the order they being approved. As a result, I have to resolve conflicts over and over... Did I miss something? )

Reply:
Hum, actually your pull requests are empty :( you should probably have rebased instead of merge:
```bash
git pull --rebase upstream master
```
Can you do a single Pull Request so that I can review them all together?

Reply:
Sure, will do in an hour, thank you.

Reply:
@gsemet, the idea was to install requirements all the time even if dags.initContainer.enabled = false.
With condition {{- if and ( .Values.dags.initContainer.enabled ) ( .Values.dags.initContainer.installRequirements ) }} and dags.initContainer.enabled = false requirement will never be installed during helm installation.

Scenario:
1. Enable dag persistence by creating PVC.
2. Put dags in PV folder along with requirements.txt
3. Install helm chart (install-requirements.sh should be executed even if dags.initContainer.enabled = false)